### PR TITLE
add: music.jp,telasa,unext,category-media-hk,jp,tw

### PR DESCRIPTION
**Pull request by shiharuharu**

### CODE REVIEW

File: .github/workflows/build.yml
Comment (IceCodeNew) [inline .github/workflows/build.yml:3]: Do one thing in a single PR.

File: data/category-media-hk
Comment (IceCodeNew) [inline data/category-media-hk:6]: please add an empty line at the end of the file.

File: data/category-media-hk
Comment (IceCodeNew) [inline data/category-media-hk:5]: separate rules in different prefixes with an extra empty line.

File: data/telasa
Comment (IceCodeNew) [inline data/telasa:4]: sort

**Review by IceCodeNew — CHANGES_REQUESTED** [review]

**Comment by IceCodeNew** [comment]
After all, I would not recommend introducing a list like `category-media-hk` or `category-media-jp`.
Just adding an attr for the sites would be much more maintainable.

Next time when you start to work on such a huge change, please try to file an issue first, state what you want, invite the maintainers to join the discussion, and wait for their approval.

File: .github/workflows/build.yml
Comment (rootmelo92118) [reply .github/workflows/build.yml:3]: Why did you want to add this setting? Which repo did you want to trigger by this workflow?

**Comment by IceCodeNew** [comment]
> @IceCodeNew If you don't recommend this operation. I think we can just close this PR.

Since the author did not respond to my comments, yep, I am going to close this PR.

**Pull request closed without merging**
